### PR TITLE
[classkit] Update for Xcode 12.5 beta 1

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

File: src/classkit.cs [hunk 1]
```diff
@@ -186,6 +186,11 @@ interface CLSActivity {
 
 		[Export ("stop")]
 		void Stop ();
+
+		[Introduced (PlatformName.MacCatalyst, 14,5)]
```
Comment (rolfbjarne) [inline src/classkit.cs:190]: You can use the short form `[MacCatalyst (14, 5)]`

**Review by rolfbjarne — APPROVED** [review]

File: src/classkit.cs (hunk 1 above)
Comment (spouliot) [reply src/classkit.cs:190]: it's a copy/paste from sharpie - making it _kind of_ shorter 😄 
but worth adding to sharpie, c.c. @dalexsoto

Comment (dalexsoto) [reply src/classkit.cs:190]: Issue'ified => https://github.com/xamarin/ObjectiveSharpie/issues/143 

I can forget 🐟🧠 but GitHub will not 🐘

**Review by tj-devel709 — APPROVED** [review]

**Comment by spouliot** [comment]
Failure is unrelated to PR -> https://github.com/xamarin/maccore/issues/2370

**Pull request merged**